ambry-cloud/src/test/resources/replicaTokens

```diff
@@ -0,0 +1,16 @@
+Version: 2
```

Comment:
These seem very specific. Should this be checked in?

Reply:
It's just a dummy test file, not even in the right format (text vs binary).